packages/wdio-junit-reporter/src/index.ts

```diff
@@ -173,6 +173,10 @@ class JunitReporter extends WDIOReporter {
                 testCase.skipped()
             } else if (test.state === 'failed') {
                 if (test.error) {
+                    if (test.error.message){
+                        test.error.message = test.error.message.replace(ansiRegex(), '')
```

Comment:
I don't think we need to create a RegExp instance everytime here. We can just put what is being returned in `ansiRegex` at the top of the file.

Reply:
done